### DISCUSSION TITLE
Shutdown docker container and resume it later

### DISCUSSION THREAD

**Discussion by leandroacostag** [mediagis/nominatim-docker#351] [Q&A]
Hello,

I have a full planet installation on an EC2 instance. Everything is working perfectly. But I'd like to shut it down and start it again in the future. If I do this will I have to rerun everything? Is it possible to resume it as it was?

On the container parameters I've specified this:
`-v nominatim-data:/var/lib/postgresql/12/main`

If it is possible, which would be the safest way of shutting it down?

**Comment by philipkozeny** [comment]
As you can see in our README you did the right thing: https://github.com/mediagis/nominatim-docker/tree/master/4.0#persistent-container-data

You have a docker volume `nominatim-data` now that hosts the PostgreSQL files. You can also verify that by running `docker volume list`.

You can just stop and restart the container, a re-import should not happen.

**Reply by leandroacostag** [reply]
Hi philip, should I start it with the same settings as the first time?

```
docker run -d -it --shm-size=64G --rm \
  -e PBF_URL=https://ftpmirror.your.org/pub/openstreetmap/pbf/planet-latest.osm.pbf \
  -e REPLICATION_URL=https://ftp5.gwdg.de/pub/misc/openstreetmap/planet.openstreetmap.org/replication/day/ \
  -e NOMINATIM_PASSWORD=XXXX \
  -e IMPORT_WIKIPEDIA=true \
  -e IMPORT_US_POSTCODES=true \
  -e IMPORT_GB_POSTCODES=true \
  -e IMPORT_TIGER_ADDRESSES=true \
  -e NOMINATIM_FLATNODE_FILE="/nominatim/flatnode.file" \
  -e POSTGRES_SHARED_BUFFERS=4GB \
  -e POSTGRES_MAINTENANCE_WORK_MEM=20GB \
  -e POSTGRES_AUTOVACUUM_WORK_MEM=4GB \
  -e POSTGRES_WORK_MEM=100MB \
  -e POSTGRES_EFFECTIVE_CACHE_SIZE=48GB \
  -e POSTGRES_MAX_WAL_SIZE=2GB \
  -e THREADS=16 \
  -v nominatim-data:/var/lib/postgresql/12/main \
  -p 8080:8080 \
  --name nominatim \
  mediagis/nominatim:4.0
```

**Reply by philipkozeny** [reply]
Yes
